src/Tests/Microsoft.Diagnostics.Monitoring.UnitTestApp/Scenarios/ExceptionsScenario.cs

```diff
@@ -57,6 +57,9 @@ public static CliCommand Command()
             reflectionTypeLoadExceptionCommand.SetAction(ReflectionTypeLoadExceptionAsync);
 
             CliCommand scenarioCommand = new(TestAppScenarios.Exceptions.Name);
+
+            scenarioCommand.SetAction(ExecuteAsync);
```

Comment:
Why is there a handler on the Exceptions command? I would think that the tests should be using one of the sub commands.